src/node/rpc/node_frontend.h

```diff
@@ -1340,8 +1357,6 @@ namespace ccf
 
       auto orc_handler = [this](auto& args, const nlohmann::json& params) {
         const auto in = params.get<ObservedReconfigurationCommit::In>();
-        LOG_DEBUG_FMT(
-          "ORC for configuration #{} from {}", in.reconfiguration_id, in.from);
 
         if (consensus == nullptr)
         {
```

Comment:
I think the consensus checks underneath are already handled by the framework (see `frontend.h`).

Reply:
I added that because it's also done in other handlers in this file, e.g. `is_primary` and `get_primary`. So, I concluded that I can't rely on the consensus being set at all time. Would this particular handler not be called at all if there's no consensus object?

Reply:
I believe the two endpoints you mentioned are very much consensus-specific and execute outside consensus (`.set_execute_outside_consensus(ccf::endpoints::ExecuteOutsideConsensus::Locally)`), hence the extra checks. In the ORC case, these can be skipped.

Reply:
That makes a lot of sense, thanks for pointing that out!